blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -664,6 +665,11 @@ def createSubMesh(blenderFacesWithRelativeVertexIndices, blenderMesh, blenderMes
 
         Face(md5SubMesh, md5VerticesOfFace[0], md5VerticesOfFace[1], md5VerticesOfFace[2])
 
+def checkPath(filePath):#python does not create directory just the file. This function added to create t eh directory

```

Comment:
In python methods get documented by adding a string as first statement in the method.

```
>>> def test(a):
...     """This is the documentation"""
...     return 5
... 
>>> test.__doc__
'This is the documentation'
```
If you use """ to specify the string you can also have a string contain line breaks.

Besides that I would suggest to use a better method name. checkPath sounds like a function that checks if a path exists and does nothing otherwise. I would suggest to name the method `createDirectoryForFileIfMissing`

Also `dir` is a build in python method and thus a different variable name should be used. The method `dir` lists what attributes an object has. So it is a very helpful method for figuring out what attributes and methods an object has